src/vlib/trace.c

```diff
@@ -346,7 +521,7 @@ cli_show_trace_buffer (vlib_main_t * vm,
 /* *INDENT-OFF* */
 VLIB_CLI_COMMAND (show_trace_cli,static) = {
   .path = "show trace",
-  .short_help = "Show trace buffer [max COUNT]",
+  .short_help = "Show trace buffer [max COUNT] [src-ip IP] [dst-ip IP]",
```

Comment:
Should the help statement here also note the `src-port` and `dst-port`?

Reply:
Yep, good catch. I'll push a fix.